b2/sync/policy.py

```diff
@@ -299,8 +297,7 @@ def make_b2_keep_days_actions(
         # Delete this version
         if deleting:
             yield B2DeleteAction(
-                dest_file.name,
-                dest_folder.make_full_path(dest_file.name), version.id_,
+                dest_file.name, dest_folder.make_full_path(dest_file.name), version.id_,
                 make_b2_delete_note(version, version_index, transferred)
```

Comment:
this gets poorly formatted due to missing , at the end of last value, please correct it